src/docs/sphinx/theory_reference/solid.rst

```diff
@@ -0,0 +1,199 @@
+   \mathbf{U} &= \left\{ \mathbf{u} \in H_1^\text{dim}(\Omega):\mathbf{u}=\mathbf{u}_D \text{ on } \Gamma_D \right\} \\
+   \hat{\mathbf{U}} &= \left\{\mathbf{u} \in H_1^\text{dim}(\Omega):\mathbf{u}=\mathbf{0} \text{ on } \Gamma_D \right\}.
+   \end{align*}
+
+and :math:`\Omega` is the current (deformed) configuration. In
+mechanics, the weak form is often referred to as the *principle of
+virtual power*. As Serac uses hyperelastic models, it is convenient to
+write this equation in the reference (undeformed) configuration
+
+.. math::
+
+   \begin{align*}
+   &\int_{\Omega_0} \sigma(\mathbf{u}) \cdot \left(\nabla_\mathbf{X} \delta \mathbf{v} \mathbf{F}^{-1} \right) \text{det}\mathbf{F}\, dV_0  - \int_{\Omega_0} \rho_0 \mathbf{b} \cdot \delta \mathbf{v} dV_0 \\ 
+   & \;\;\;\;\;\;\;\;\;\;\;\; - \int_{\Gamma_{N_0}} \delta\mathbf{v}\cdot \mathbf{t}\,||\mathbf{F}^{-T}\mathbf{n}_0||\text{det}\mathbf{F}\, dA_0 + \int_{\Omega_0} \rho_0\ddot{\mathbf{u}} \cdot \delta\mathbf{v}\, dV_0= 0, & & \forall \delta\mathbf{v} \in \hat{\mathbf{U}}
+   \end{align*}
+
+where :math:`\nabla_X` is the gradient with respect to the reference (material) coordinates.
+
+Material Models
+===============
+
+Serac uses *hyperelastic* material formulations, i.e. materials that behave elastically under large deformations. Mathematically,
+this implies they are derived from a *strain energy density* function :math:`W=W(\mathbf{F})`. It can be shown that
+
+.. math::
+
+   \sigma(\mathbf{F}) = \frac{1}{\text{det}\mathbf{F}} \frac{\partial W}{\partial \mathbf{F}} \mathbf{F}^T  = \frac{1}{\text{det}\mathbf{F}} \mathbf{P}  \mathbf{F}^T
+
+where
+
+.. math::
+
+   \mathbf{P} =  \frac{\partial W}{\partial \mathbf{F}} = {\text{det}}\mathbf{F} \sigma \mathbf{F}^{-T}
+
+is the *first Piola-Kirchhoff stress*. Serac currently only has two material models. First, a neo-Hookean material where
+   
+.. math::
+
+   \begin{align*}
+   W(\mathbf{F}) &= \frac{\mu}{2}(\bar{I}_1 - \text{dim}) + \frac{K}{2}(\text{det}\mathbf{F} - 1)^2 \\
+   \bar{I}_1 &= \frac{\text{trace}(\mathbf{F}\mathbf{F}^T)}{(\text{det}\mathbf{F})^{2/\text{dim}}}
+   \end{align*}
+
+and :math:`\mu` and :math:`K` are the shear and bulk modulus, respectively. This definition also
+implies that the 2D simulations are using a plane strain assumption. The second model is a small strain isotropic linear elastic material where
+
+.. math::
+
+   \begin{align*}
+   \sigma(\epsilon) &= \lambda \text{trace}(\epsilon) \mathbf{I} + 2\mu \epsilon \\
+   \epsilon &= \frac{1}{2}\left(\mathbf{F} + \mathbf{F}^T \right) - \mathbf{I} \\
+   \lambda &= K - \frac{2}{\text{dim}} \mu
+   \end{align*}
+
+and :math:`\epsilon` is the linearized strain tensor. Note that this model is only valid for small strains 
+where the neo-Hookean model is nearly equivalent. It is included mostly for testing purposes.
+
+Optionally, we can also model linear viscoelasticity by adding
+
+.. math::
+
+   \sigma(\dot{\epsilon}) = \eta \dot{\epsilon}
```

Comment:
"Optionally, we can add a Kelvin-Voigt linear viscoelastic term by adding"

Note that usually that would be something like \sigma(\dot{epsilon}) = \eta C \dot{\epsilon}

e.g. the viscous coefficient multiplies the elasticity tensor, so it has units of time

Reply:
This is me reusing the existing MEFM machinery. I think when Sam's stuff comes online, we should move to this formulation as it is more natural.